package.json

```diff
@@ -68,5 +68,9 @@
       "unlabeled": ":question: Unlabeled"
     },
     "wildcardLabel": "unlabeled"
+  },
+  "volta": {
+    "node": "20.14.0",
+    "pnpm": "8.15.9"
```

Comment:
matches `packageManager`

Reply:
tbh, volta should probably just read packageManager and .npmrc, and only _optionally_ have the volta package.json thing